src/styles/blocks/catalog.scss

```diff
@@ -11,6 +11,7 @@
     justify-content: center;
     align-items: center;
     box-sizing: border-box;
+    transition: transform 300ms;
```

Comment:
should we change color with transition as well?

<img width="492" alt="Screenshot 2023-02-28 at 10 45 36 AM" src="https://user-images.githubusercontent.com/60295799/221800426-d85de944-fbca-457b-be54-b41b8044be0e.png">